src/App.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import styles from './App.module.scss';
+import { ErrorButton } from './components/ErrorButton/ErrorButton';
+import { ResultContainer } from './components/ResultContainer/ResultContainer';
+import { ResultResponse } from './types';
+
+export class App extends Component {
+  state = {
+    searchValue: this.setSearchValue(),
+    results: [],
+    isLoading: false,
+  };
+
+  setSearchValue() {
+    const searchKeyWord = localStorage.getItem(
+      'lev1ossa-react-components-value'
+    );
+    return !!searchKeyWord ? searchKeyWord : '';
+  }
+
+  searchButtonHandler = () => {
```

Comment:
as your button is type 'submit', you should use preventDefault or change it to 'button' to avoid unnecessary redirection and this 
![image](https://github.com/Lev1ossa/RSS_React/assets/89910618/17bf9048-9312-48b8-be58-f3b3e568590a)